### ISSUE TITLE
Merged default-disabled original objects aren't enabled when toggled on

### ISSUE THREAD

**Issue by lackofbindings** [d4rkc0d3r/d4rkAvatarOptimizer#133]
I have a few clothing items as separate skinned meshes on my avatar. However each item also has nested physbones and other components under each one. Normally this makes it easy to toggle all related components as one. When being merged by the optimizer they are converted to NaNimation, but it seems like the keyframes that enable the original gameobjects are removed by the optimizer so the gameobjects stay off and thus so do the physbones. This avatar is WD-Off in case it matters.

Please keep intact the enabling/disabling of the original objects when merging, if that is the case.

**Comment by d4rkc0d3r** [comment]
Yeah thats an issue thats been bugging me for a while. The reason that it still isn't fixed is that I don't create a new mesh renderer and rather merge them all into the first one of each group. Therefore that gameobject has to be always on and can't keep the toggle anims.

I definitely want to fix this, I just am not quite sure how yet.
